### PR TITLE
Edits to YAMLs

### PR DESCRIPTION
**Pull request by kyessenov**
Some minor edits. Feel free to close and apply in your PR.

---
This change is [Reviewable](https://reviewable.io/reviews/istio/istio/30)

### CODE REVIEW

File: demos/apps/bookinfo/bookinfo-istio.yaml [hunk 1]
```diff
@@ -127,12 +127,14 @@ spec:
...
         - containerPort: 9080
       - name: proxy
-        image: docker.io/istio/runtime:20170224214559
+        image: docker.io/istio/runtime:a87bea5
         securityContext:
           runAsUser: 1337
         args:
           - proxy
           - sidecar
+          - -n
```
Comment (andraxylia) [inline demos/apps/bookinfo/bookinfo-istio.yaml]: Is there an API for proxy to obtain the namespace in which it is running or can we pass the namespace via env vars?

Comment (andraxylia) [inline demos/apps/bookinfo/bookinfo-istio.yaml]: What is proxy is this yaml is deployed in a different namespace? Please remove this.

Comment (kyessenov) [inline demos/apps/bookinfo/bookinfo-istio.yaml]: This restricts traffic to a namespace and helps to reduce load on the controller and the restart frequency. We can add another flag to use the current namespace, which I like.

**Comment by frankbu** [comment]
Closing this one. I'll incorporate these changes in another PR, where I will also updates the demo to work with the fault-rule design change, i.e., set the fault (delay) in a route-rule instead of destination-policy.

**Pull request closed without merging**